Lib/unittest/mock.py

```diff
@@ -1469,13 +1474,51 @@ def get_original(self):
             )
         return original, local
 
+    @property
+    def is_started(self):
+        return self._context is not None
+
+    @property
+    def is_local(self):
+        return self._context.is_local
+
+    @is_local.setter
+    def is_local(self, value):
+        self._context.is_local = value
+
+    @property
+    def original(self):
+        return self._context.original
+
+    @original.setter
+    def original(self, value):
+        self._context.original = value
```

Comment:
Actually when I said to rename it to `original`, it was only for the `context`'s field, not for the patch object itself!

For the patch's object, I think we can keep `temp_original` (or maybe you wanted to have the same name for both the context dataclass and the patch object? if so, sorry for the back-and-forth)

Reply:
Agreed, let's keep `temp_original` in `_patch` while renaming it for `._context`